### DISCUSSION TITLE
Is microstream REST Interface integrated in helidon?

### DISCUSSION THREAD

**Discussion by dcoracle** [microstream-one/microstream#302] [Q&A] [answered]
Hi,

documentation is very sparse but is microstream REST API and Client GUI natively integrated in helidon yet?  I wasn't able to find the proper classes (StorageRestService) when trying to set it up.  I am using helidon 2.4.0

Thanks,
Derek

**Comment by hg-ms** [comment, marked as answer]
Hello,

the REST API and the Client GUI are not integrated into Helidon.

Best Regards
